.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,35 +1,41 @@
 <!--
-    If you have any questions regarding your scenario or components behavior, please provide a detailed explanation.
-    If you own an active DevExtreme license, feel free to contact us in the Support Center (https://www.devexpress.com/ask) to receive an answer shortly.
+    We cannot guarantee a quick answer on GitHub.
+    If you own an active DevExtreme license, feel free to contact us in the Support Center (https://www.devexpress.com/ask)
+    to receive an answer shortly.
 -->
 
 <!--
-    Checked checkbox should look like this: [x]
+    A checked check box should look like this: [x]
 -->
 - [ ] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.
```

Comment:
and believe?

Reply:
> [believe](https://www.macmillandictionary.com/dictionary/british/believe#believe__1)
[transitive] to think that a fact is true

Reply:
I know what believe means, however, 'believe' is a matter of perspective or opinion.

Reply:
That is correct. The person that opens an issue is of opinion that he or she does not create a duplicate. However, it might be not true.